packages/styles/scss/patterns/card-section/_card-section.scss

```diff
@@ -63,6 +65,7 @@
 
       &:hover {
         cursor: pointer;
+        background: $hover-ui;
```

Comment:
Shouldn't this live in the theme mixin above?

Reply:
In general I'm seeing interaction issues with the cards themselves.

Reply:
@photodow: you mean the hover class could be outside of mixin?

Reply:
And interactions issues means like flickering while switching b/w cards?

Reply:
Low quality `gif`, but I'm not seeing any interaction. `:hover`, `:active`, `:focus` all seems off or none existent. It should just come in default via the `CardLink`, so I'm wondering if we overrode everything on accident.

![Jan-07-2020 13-45-16](https://user-images.githubusercontent.com/3793636/71924126-fd4dc380-3153-11ea-91ba-b2acfef64081.gif)